.gitignore

```diff
@@ -1,7 +1,7 @@
 .buildlog
 .DS_Store
 .idea
-.pub/
+.dart_tool
```

Comment:
`.dart_tool/` - it'll just match the directory in the project root

Reply:
👍